internal/build/env.go

```diff
@@ -88,11 +89,16 @@ func LocalEnv() Environment {
 			env.Branch = b
 		}
 	}
-	// Note that we don't get the current git tag. It would slow down
-	// builds and isn't used by anything.
+	if env.Tag == "" {
+		env.Tag = firstLine(RunGit("for-each-ref", "--points-at", "HEAD", "--format=%(refname:short)", "refs/tags"))
```

Comment:
The `for-each-ref` has a `count` argument so you can limit with that (`--count 1`) instead of having to do this `firstLine` magic.